backend/src/models/acronym.js

```diff
@@ -21,7 +21,12 @@ module.exports = (sequelize, DataTypes) => {
     name: {
       allowNull: false,
       comment: 'Name of the application acronym',
-      type: DataTypes.STRING(64)
+      type: DataTypes.STRING(120)
```

Comment:
No, I just thought of that as well actually.
I'll need a migration to widen that field. Can add that to the next PR.

Reply:
It would be better to have that as part of this PR - it's either that or hold off on the string size extension here and put that for the next PR. It's not great practice to do a model change without its associated migration in separate PRs.

Reply:
At the end of the day it's not going to really matter since I'm doing 2 back to back PRs for this feature but yes it's cleaner to have the model change and the script in the same one so I'll do that.